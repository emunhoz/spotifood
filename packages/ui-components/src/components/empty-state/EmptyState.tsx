import React from 'react'
import SearchIcon from './image/search.svg'
import * as S from './EmptyState.style'

interface IEmptyState {
  title?: string
  message?: string
}

export const EmptyState = ({ title = '', message = '' }: IEmptyState) => (
  <S.EmptyState>
    <S.EmptyStateImage src={SearchIcon} alt={title} />
    <S.EmptyStateTitle>{title}</S.EmptyStateTitle>
    <S.EmptyStateMessage>{message}</S.EmptyStateMessage>
  </S.EmptyState>
)
