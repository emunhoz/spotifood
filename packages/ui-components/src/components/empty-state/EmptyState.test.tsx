import React from 'react'
import { render } from '../../utils/test-utils'
import { EmptyState } from './EmptyState'

describe('[Component]: EmptyState', () => {
  test('render the component', () => {
    const { asFragment } = render(<EmptyState />)
    expect(asFragment()).toMatchSnapshot()
  })
})