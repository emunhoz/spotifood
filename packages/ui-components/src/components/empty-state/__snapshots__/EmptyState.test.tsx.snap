// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: EmptyState render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM deDEVT"
  >
    <img
      alt=""
      class="sc-dlnjwi fxotzT"
      src="test-file-stub"
    />
    <h2
      class="sc-gtsrHT kEeQDH"
    />
    <p
      class="sc-hKFxyN FKMGg"
    />
  </div>
</DocumentFragment>
`;
